packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -329,29 +340,28 @@ class FlexLayoutProperties extends LayoutProperties {
   }) : super(node);
 
   factory FlexLayoutProperties.fromDiagnostics(RemoteDiagnosticsNode node) {
-    if (node == null) return null;
     // Cache the properties on an expando so that local tweaks to
     // FlexLayoutProperties persist across multiple lookups from an
     // RemoteDiagnosticsNode.
-    return _flexLayoutExpando[node] ??= _buildNode(node);
+    return (_flexLayoutExpando[node] ??= _buildNode(node))!;
```

Comment:
what if _buildNode(node) returns null

Reply:
moved null-check to _buildNode, where it seems to be better fit